src/anbox/cmds/launch.cpp

```diff
@@ -69,51 +86,95 @@ anbox::cmds::Launch::Launch()
     auto bus = std::make_shared<core::dbus::Bus>(core::dbus::WellKnownBus::session);
     bus->install_executor(core::dbus::asio::make_executor(bus, rt->service()));
 
+    const auto snap_path = utils::get_env_value("SNAP");
+    if (!snap_path.empty()) {
+      const auto resource_path = utils::string_format("%s/usr/share/anbox", snap_path);
```

Comment:
I would rather prefer const auto resource_path = fs::path(snap_path) / "usr" / "share" / "anbox"; with fs begin shorthand for boost::filesystem.